src/communication/request_client_server.hpp

```diff
@@ -337,16 +337,18 @@ namespace cubcomm
   css_error_code request_server<MsgId>::receive_request_buffer (std::unique_ptr<char[]> &message_buffer,
       size_t &message_size)
   {
-    int ilen = 0;
-    css_error_code err = m_channel.recv_int (ilen);
+    size_t expected_size = 0;
+    std::size_t size_ilen = sizeof (expected_size);
```

Comment:
Nit:
I think there is no difference between std::size_t and size_t. How about unifying it as size_t?

Reply:
Done